kodilla-stream/src/main/java/com/kodilla/stream/StreamMain.java

```diff
@@ -1,8 +1,23 @@
 package com.kodilla.stream;
 
+import com.kodilla.stream.forumuser.Forum;
+import com.kodilla.stream.forumuser.ForumUser;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
 public class StreamMain {
 
     public static void main(String[] args) {
-
+        Forum forum = new Forum(20);
+        Map<String, ForumUser> forumUsers = forum.getUsersList()
+                .stream().filter(forumUser -> forumUser.getGender() == 'M')
+                .filter(forumUser -> forumUser.getDateOfBirth().getYear() <= 2001)
```

Comment:
dzis jest 17.07.2021. Jesli ktos jest urodzony 19.07.2001 to nie ma dzis jeszcze 20 lat